geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -4,39 +4,77 @@ use geo_types::coord;
 
 use crate::{Coord, EuclideanLength, GeoFloat, Line, MultiPoint, Point, Polygon, Rect, Triangle};
 
-use crate::triangulate_delaunay::{
-    DelaunayTriangle, DelaunayTriangulationError, DEFAULT_SUPER_TRIANGLE_EXPANSION,
-};
+use crate::triangulate_delaunay::{DelaunayTriangle, DelaunayTriangulationError};
 use crate::{BoundingRect, TriangulateDelaunay};
 
 type Result<T> = result::Result<T, VoronoiDiagramError>;
 
+const DEFAULT_CLIPPING_MASK_EXPANSION: f64 = 20.;
+
+/// The default value for determining if the slope of a line is
+/// near zero or near infinite.
+///
+/// A near infinite slope is greater than this value.
+///
+/// A near zero slope is less than `1 / DEFAULT_SLOPE_THRESHOLD`.
 pub const DEFAULT_SLOPE_THRESHOLD: f64 = 1e3;
 
+/// The components of a Voronoi Diagram.
 #[derive(Debug, Clone)]
 pub struct VoronoiComponents<T: GeoFloat> {
+    /// The Delaunay Triangles used to construct the Voronoi diagram.
     pub delaunay_triangles: Vec<Triangle<T>>,
+    /// The vertices of the Voronoi diagram.
     pub vertices: Vec<Coord<T>>,
+    /// The lines of the Voronoi diagram.
     pub lines: Vec<Line<T>>,
+    /// This is an index of Delaunay triangles that are / have neighours
+    /// the first value is the index of the triangle and the second value
+    /// is the index of a neighbouring triangle.
+    /// If the triangle does not have a neighbour the value is `None`.
     pub neighbours: Vec<(Option<usize>, Option<usize>)>,
```

Comment:
IMO, for the first cut, `neighbours` shouldnt be `pub` .  i.e. `Vec<(Option<usize>, Option<usize>)>` isnt a type that should be part of the public contract.  If someone else wants to use `neighbours` outside of geo, let them do the work of designing an API that can be supported even if the implementation changes.